pkg/cluster/member.go

```diff
@@ -15,28 +15,31 @@
 package cluster
 
 import (
-	"errors"
-
 	"github.com/coreos/etcd-operator/pkg/spec"
 	"github.com/coreos/etcd-operator/pkg/util/etcdutil"
 
 	"k8s.io/kubernetes/pkg/api"
 )
 
-var errMemberNotReady = errors.New("etcd member's Name is empty. It's not ready yet.")
-
 func (c *Cluster) updateMembers(known etcdutil.MemberSet) error {
 	resp, err := etcdutil.ListMembers(known.ClientURLs())
 	if err != nil {
 		return err
 	}
 	members := etcdutil.MemberSet{}
```

Comment:
can we move line29 to line58 into etcdutil (from resp to memberSet)? and add a unit test for it?